hw6/assets/style.css

```
@@ -0,0 +1,254 @@
+@font-face {
+    font-family: 'Roboto';
+    src: url('./fonts/Roboto-Regular.ttf') format('truetype');
+    font-weight: 400;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'Roboto';
+    src: url('./fonts/Roboto-Bold.ttf') format('truetype');
+    font-weight: 500;
+    font-style: normal;
+}
+
+/* Tmpl main styles */
+
+body {
+    font-family: 'Roboto', sans-serif;
+}
+
+section, header, footer {
+    width: 100%;
+}
+
+header {
+    background: #EEEFF1;
+    position: sticky;
+    top: 0;
+}
+
+footer {
+    background: #34547A;
+}
+
+.navigation, .wrapper {
+    display: block;
+    width: 100%;
+    max-width: 1120px;
+    margin: 0 auto;
+}
+
+.container {
+    display: block;
+    width: 100%;
+    max-width: 540px;
+}
+
+/* Nav styles */
+
+.navigation {
+    min-height: 80px;
+    position: relative;
+}
+
+.navigation > * {
+    position: absolute;
+    bottom: 0;
+}
+
+.navigation.sticky {
+    background-color: white;
+    max-width: unset;
+    box-shadow: 0 4px 6px -1px rgb(0 0 0 / 0.1), 0 2px 4px -2px rgb(0 0 0 / 0.1);
+}
+
+.navigation.sticky > * {
+    top: 30px;
+    display: inline-block;
+    left: 37.5%;
+    transform: translateX(-50%);
+    margin: 0 auto;
+}
+
+.navigation ul li {
+    display: inline-block;
+    white-space: nowrap;
+}
+
+.navigation ul li a {
+    color: #000;
+    font-size: 16px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 164.063%;
+    letter-spacing: 1.6px;
+    text-transform: uppercase;
+    text-decoration: none;
+}
+
+.navigation ul li:not(:first-child) a {
+    margin-left: 42px;
+}
+
+.navigation ul li a:hover, .navigation ul li a:focus {
```

так же, лучше в 2 линии
```suggestion
.navigation ul li a:hover,
.navigation ul li a:focus {
```